docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -285,37 +285,44 @@ The master supports the following configuration settings:
 
       -  ``job_storage_root``: The shared directory where job-related files will be stored. It is
          where the needed Determined executables are copied to when the experiment is run, as well
-         as where the Slurm ``sbatch`` script and log files are created. This directory must be
-         visible to the launcher and from the compute nodes.
+         as where the Slurm/PBS scripts and log files are created. This directory must be writable
+         by the launcher and accessible to the compute nodes to enable the logs to be written from
```

Comment:
Not sure of "enable the logs from the submitted batch jobs to be written" sounds better.